### ISSUE TITLE
Contact Form: issue typing into "Unique CSS ID" field for selects and checkbox groups

### ISSUE THREAD

**Issue by tbradsha** [Automattic/jetpack#23100]
### Impacted plugin

Jetpack

### Steps to Reproduce

1. Add a contact form block to a page.
2. Add a new "select" block to the contact form.
3. Add a few options (the issue only occurs if there are items here).
4. In the block settings, open Advanced and attempt to type in the "Unique CSS ID" field.

Do the same with a "checkbox group" block. I suspect it affects any multiple-option block.

### A clear and concise description of what you expected to happen.

I should be able to type text into the "Unique CSS ID" field.

### What actually happened

After one keypress, focus is moved away from the "Unique CSS ID" field and into the last option in the affected block.

### Other information

https://user-images.githubusercontent.com/32492176/155600768-e2e9aaa7-75a4-45b3-8e11-dc30df5f3638.mov

### Operating System

macOS

### OS Version

12.1

### Browser

Chrome/Chromium

### Browser Version(s)

98.0.4758.102

**Comment by ntsekouras** [comment]
Closing as this `Unique CSS ID` doesn't seem to exist now. Feel free to reopen if I'm missing something.

**Comment by tbradsha** [comment]
Indeed, it looks like this is no longer present, probably as of #30260.
